geps/gep-713.md

```diff
@@ -560,50 +560,536 @@ ties:
+
+**Charlie**: _At a guess, all the workloads in the `baker` namespace actually
+fail a lot, but they seem OK because there are retries across the whole
+namespace?_ 🤔
+
+Ana's blood runs cold.
+
+**Charlie**: _Yeah. Looking a little closer, I think your `baker` rollout this
+morning would have failed without those retries._ 😕
+
+There is a pause while Ana's mind races through increasingly unpleasant
+possibilities.
+
+**Ana**: _I don't even know where to start here. How long did that
+RetryPolicy go in? Is it the only thing like it?_
+
+**Charlie**: _Didn’t look closely before deleting it, but I think it said a few
+months ago. And there are lots of different kinds of policy and lots of
+individual policies, hang on a minute..._
+
+**Charlie**: _Looks like about 47 for your chunk of the world, a couple hundred
+system-wide._
+
+**Ana**: 😱 _Can you tell me what they’re doing for each of our services? I
+can’t even_ look _at these things._ 😕
+
+**Charlie**: _That's gonna take awhile. Our tooling to show us which policies
+bind to a given workload doesn't go the other direction._
+
+**Ana**: _...wait. You have to_ build tools _to know if retries are turned on??_
+
+Pause.
+
+**Charlie**: _Policy attachment is more complex than we’d like, yeah._ 😐
+_Look, how ‘bout roll back your `baker` change for now? We can get together in
+the morning and start sorting this out._
+
+Ana shakes her head and rolls back her edits to the `baker` Deployment, then
+sits looking out over the lake as the deployment progresses.
+
+**Ana**: _Done. Are things happier now?_
+
+**Charlie**: _Looks like, thanks. Reckon you can get back to your sailboard._ 🙂
+
+Ana sighs.
+
+**Ana**: _Wish I could. Wind’s died down, though, and it'll be dark soon.
+Just gonna head home._
+
+**Charlie**: _Ouch. Sorry to hear that._ 😐
+
+One more look out at the lake.
+
+**Ana**: _Thanks for the help. Wish we’d found better answers._ 😢
+
+### The Problem, restated
+What this parable makes clear is that, in the absence of information about what
+Policy is affecting an object, it’s very easy to make poor decisions.
+
+It’s critical that this proposal solve the problem of showing up to three things,
+listed in increasing order of desirability:
+- _That_ some Policy is affecting a particular object
+- _Which_ Policy is (or Policies are) affecting a particular object
+- _What_ settings in the Policy are affecting the object.
+
+In the parable, if Ana and Charlie had known that there were Policies affecting
+the relevant object, then they could have gone looking for the relevant Policies
+and things would have played out differently. If they knew which Policies, they
+would need to look less hard, and if they knew what the settings being applied
+were, then the parable would have been able to be very short indeed.
+
+(There’s also another use case to consider, in that Charlie should have been able
+to see that the Policy on the namespace was in use in many places before deleting
+it.)
```

Comment:
Shouldn't that be a default assumption if there are other resources in the namespace?

Reply:
Yes, but again, it's easy to miss in a stressful situation. Explicitly _telling_ people this is better than leaving it implicit.